### ISSUE TITLE
Flood protection

### ISSUE THREAD

**Issue by matschaffer** [ekmartin/slack-irc#21]
While I haven't hit this yet (just started running this today) - I noticed some folks on the web talking about issues with flood protection when mirroring slack & irc.

The irc module [has support for this](https://github.com/martynsmith/node-irc/blob/605555e3dfe5eb92070f4520a22cedfbebd4dca8/lib/irc.js#L52) so it might be nice to have a way to pass the flag down from the slack-irc config to make use of it.

Thanks!

**Comment by ekmartin** [comment]
Hm, that's probably smart. The default delay of 1000 ms between messages is quite high though, it'll probably make the chat feel less fluid. Maybe turning it down to 500 ms could work. I think that at least networks like Freenode allow short burts of messages without kicking.

**Comment by ekmartin** [comment]
Flood protection is now on by default, fixed in version [3.2.0](https://github.com/ekmartin/slack-irc/blob/master/CHANGELOG.md#320---2015-04-03). You can now also pass options directly to node-irc, like this:
```js
    "ircOptions": { // Optional node-irc options
      "floodProtection": false, // On by default
      "floodProtectionDelay": 1000 // 500 by default
    }
```

**Comment by matschaffer** [comment]
Awesome, thanks!
